charts/azure-apps/examples/secrets-store-csi-driver-provider-azure.yaml

```diff
@@ -0,0 +1,9 @@
+azureKvCsiProvider:
```

Comment:
should we also rename this files to match `azure-kv-csi-provider` ?

Reply:
Yes, why not, to be consistent, thanks for the suggestion